frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -367,6 +374,26 @@ local footerTextGeneratorMap = {
             end
         end
     end,
+    page_turning_inverted = function(footer)
+        local symbol_type = footer.settings.item_prefix
+        local prefix = symbol_prefix[symbol_type].page_turning_inverted
+        if G_reader_settings:isTrue("input_invert_page_turn_keys") or G_reader_settings:isTrue("input_invert_left_page_turn_keys") or
+           G_reader_settings:isTrue("input_invert_right_page_turn_keys") or G_reader_settings:isTrue("inverse_reading_order") then
+            if symbol_type == "icons" or symbol_type == "compact_items" then
+                return symbol_prefix.icons.page_turning_inverted
+            else
+                return T(_("%1".. "I"), prefix)
+            end
+        elseif footer.settings.all_at_once and footer.settings.hide_empty_generators then
+            return ""
+        else
+            if symbol_type == "icons" or symbol_type == "compact_items" then
+                return symbol_prefix.icons.page_turning_regular
+            else
+                return T(_("%1".. "R"), prefix)
```

Comment:
I don't think it's a good idea to use "R" for regular. R is very commonly used for "reversed" and could likely cause confusion in this context. I would suggest N for normal/neutral.